web/gen/options_js.py

```diff
@@ -49,11 +50,14 @@ async def make() -> str:
         print("")
         print("export const defaultState: OptionsState = {")
         for _, opt in sorted(m.options.items()):
-            print(
-                f"    {opt.name}: {json.dumps(opt.default)},".replace(
-                    ": null", ": undefined"
+            if opt.name == "web_columns":
+                print(f"    {opt.name}: {json.dumps(DEFAULT_WEB_COLUMNS)},")
+            else:
+                print(
+                    f"    {opt.name}: {json.dumps(opt.default)},".replace(
+                        ": null", ": undefined"
+                    )
```

Comment:
This needs to go unless there's justification for it. You didn't address my previous comment.

Reply:
I didn't know the default values were hardcoded in the `webaddons.py` so I added If else to check to fill the web_columns with the default value. Now I will remove this 😊